akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -302,6 +301,8 @@ private[remote] abstract class ArteryTransport(_system: ExtendedActorSystem, _pr
   import ArteryTransport._
   import FlightRecorderEvents._
 
+  type LifeCycle
```

Comment:
internal api?

Reply:
The whole type is Internal API and private to remote